README.md

```diff
@@ -52,7 +52,7 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
 
 1. Bayesian Classification (bayes)
 
-    This workload benchmarks NaiveBayesian Classification implemented in Spark-MLLib/Mahout examples.
+    This workload benchmarks NaiveBayesian Classification implemented in Spark-MLLib/Mahout.
```

Comment:
why do you leave `Mahout` here?

Reply:
datagen is from mahout

Reply:
thanks for the explanation, but we don't want to confuse the end user right?

Reply:
ok, I will remove mahout.